resident/resident-service/src/main/java/io/mosip/resident/constant/ResidentErrorCode.java

```diff
@@ -113,7 +113,10 @@ public enum ResidentErrorCode {
 	OTP_REQUEST_FLOODED("RES_SER_510", "Innumerous OTP requests received"),
 	EID_NOT_BELONG_TO_SESSION("RES-SER-511", "The entered EID is not associated with the UIN/VID used to log in."),
 	GRIEVANCE_TICKET_GENERATION_FAILED("RES-SER-512", "Unable to Generate Ticket of grievance"),
-	MISSING_INPUT_PARAMETER("RES-SER-414", "Missing input Parameter- %s");
+	MISSING_INPUT_PARAMETER("RES-SER-414", "Missing input Parameter- %s"),
+	VID_CREATION_FAILED("RES-SER-513",
+			"This VID cannot be revoked since this VID was used to log into the current session.");
```

Comment:
Correct the message:
Failed to create VID, as the existing VID cannot be revoked since this VID was used to log into the current session.

Reply:
done.